src/libaktualizr/storage/invstorage.h

```diff
@@ -145,11 +149,13 @@ class INvStorage {
   virtual void storeInstallationResult(const data::OperationResult& result) = 0;
   virtual bool loadInstallationResult(data::OperationResult* result) = 0;
   virtual void clearInstallationResult() = 0;
+  virtual boost::optional<std::pair<int64_t, size_t>> isTargetFileExists(const Uptane::Target& target) const = 0;
```

Comment:
Not a big deal, but would it be better if this were named something like `checkTargetFile`?